extensions/rich_text_components/Image/Image.py

```diff
@@ -26,6 +26,7 @@ class Image(base.BaseRichTextComponent):
     frontend_name = 'image'
     tooltip = 'Insert image'
     requires_fs = True
+    preview_url_to_interpolate = '/imagehandler/<[explorationId]>/<[filepath]>'
```

Comment:
Where is 'filepath' supplied? I don't see it in the dict passed to $interpolate.